.circleci/config.yml

```diff
@@ -57,6 +57,42 @@ jobs:
           name: Scan dependencies for vulnerabilities
           working_directory: ~/workbench/api
           command: ./project.rb gradle dependencyCheckAnalyze --info
+  api-build-integration-test:
+    docker:
+      - image: allofustest/workbench:buildimage-0.0.10
+    working_directory: ~/workbench
+    environment:
+      JVM_OPTS: -Xmx3200m
+      TERM: dumb
+    steps:
+      - checkout
+      - run:
+          name: Fetch Submodules
+          command: git submodule update --init
+      - restore_cache:
+          keys:
+          - api-cache-{{ checksum "~/workbench/api/build.gradle" }}
```

Comment:
nit: Should probably use a different cache key than the one above, e.g. api-integration-cache-... the exploded API stuff won't be cached here so that might cause issues